aries_vcx/src/protocols/proof_presentation/verifier/state_machine.rs

```diff
@@ -505,6 +519,50 @@ pub mod unit_tests {
         }
     }
 
+    mod build_messages {
+        use crate::messages::a2a::MessageId;
+        use crate::messages::proof_presentation::presentation_request::PresentationRequestData;
+        use crate::protocols::proof_presentation::verifier::state_machine::{
+            build_starting_presentation_request, build_verification_ack,
+        };
+        use crate::utils::devsetup::{was_in_past, SetupMocks};
+
+        #[test]
+        #[cfg(feature = "general_test")]
+        fn test_verifier_build_verification_ack() {
+            let _setup = SetupMocks::init();
+
+            let msg = build_verification_ack("12345");
+
+            assert_eq!(msg.id, MessageId("testid".into())); // todo: it should generate random uuid even in test
```

Comment:
Now I finally understand your goal with these thread id comments, but still don't see much benefit in that.